outputs/docker.go

```diff
@@ -0,0 +1,108 @@
+package outputs
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/aquasecurity/postee/v2/layout"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/stdcopy"
+	"github.com/google/uuid"
+)
+
+type DockerClient struct {
+	client  client.APIClient
+	uuidNew func() uuid.UUID
+
+	Name      string
+	ImageName string
+	Cmd       []string
+	Volumes   map[string]string
+	Env       []string
+}
+
+func (d DockerClient) GetName() string {
+	return d.Name
+}
+
+func (d *DockerClient) Init() error {
+	var err error
+	d.client, err = client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	if err != nil {
+		return fmt.Errorf("failed to initialize docker action: %w", err)
+	}
+	d.uuidNew = uuid.New
+
+	log.Println("docker action successfully initialized")
+	return nil
+}
+
+func (d DockerClient) Send(m map[string]string) error {
+	ctx := context.Background()
+
+	_, err := d.client.ImagePull(ctx, d.ImageName, types.ImagePullOptions{})
```

Comment:
ignored argument is `io.ReadCloser` instance which needs to be closed explicitily

Reply:
thanks, good catch - fixed here https://github.com/aquasecurity/postee/pull/312/commits/18b6760015d8d91248985c9268cc57cfbada1d08